### ISSUE TITLE
bug in comments inside the lambda expression

### ISSUE THREAD

**Issue by rihou** [INRIA/spoon#2763]
The comments in the lambda expression are reported to have errors.

An example code:
```java
import java.util.Arrays;
import java.util.List;
import java.util.stream.Collectors;

public class CommentInLambdaExpressionTest {

public static void main(String[] args) {
	CommentInLambdaExpressionTest test = new CommentInLambdaExpressionTest();
	List<Integer> lst = test.setup();
	System.out.println(lst);
}

public List<Integer> setup() {
	List<Integer> messages = Arrays.asList(1, 2, 3, 4, 5, 6, 7, 8, 9, 10);
	return messages.stream().filter( num -> 
	// comment in lambda expression
	num > 5
	).collect(Collectors.toList());
}
}
```
The code triggers the error:
`2018-11-08 10:11:15,847 ERROR spoon.support.compiler.jdt.JDTCommentBuilder - "// comment in lambda expression" cannot be added into the AST, with parent class spoon.support.reflect.code.CtLambdaImplplease report the bug by posting on #2482`

**Comment by pvojtechovsky** [comment]
Thank you for reporting 👍

**Comment by monperrus** [comment]
closed by #2801
